definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -6333,7 +6291,7 @@
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "boRequestTranslationFromWLU",
     "UserRole": "caseworker-divorce-solicitor",
-    "CRUD": "CRU"
```

Comment:
Just need to keep this original object in "AuthorisationCaseEvent-prod.json".

Ideally, **if** Jeremy confirms every role should have "R" access. it would be to have the following changes:
inside prod file:
- original authorisations
inside nonprod file:
- caseworker-divorce-solicitor role with R access